### PR TITLE
Ford sam

### PR DESCRIPTION
**Pull request by FordSam**

### CODE REVIEW

File: index.md
Comment (liaosunny123) [inline index.md:1]: You can not delete this file

File: FordSam/fork.txt
Comment (liaosunny123) [inline FordSam/fork.txt:1]: This file carries unright commit information, please try to close this pr and open another one using right commit information

**Review by liaosunny123 — CHANGES_REQUESTED** [review]
please try to finish the conversation

File: FordSam/fork.txt
Comment (FordSam) [reply FordSam/fork.txt:1]: Sorry,I think my answer is correct.Can you make my mistake a little clearer？

Comment (liaosunny123) [reply FordSam/fork.txt:1]: > Sorry,I think my answer is correct.Can you make my mistake a little clearer？

Your file and forkme file are both right, however, your github action have some mistakes.
The mistakes follows:
- You can not delete the README.md file.
- You should copy the `forkme.txt` but not move the whole directory, you can find that you push my `init` commit also, you should only push your own commit log.

How can you solve it:
Try to delete your fork repo and fork again. Then clone it and try to do it from the start.

Comment (FordSam) [reply FordSam/fork.txt:1]: Get

**Pull request closed without merging**